CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Added
+
+- Typescript Support (TODO: Add link to final pull request typescript -> master)
```

Comment:
This line will need to be updated during the next release:

Typescript Support ([#42](https://github.com/screepers/screeps-server-mockup/pull/42))

Reply:
Will do